From a7af22e6ad008c6760eed4ea60691f864c4712f7 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 6 May 2021 21:20:58 +0530
Subject: [PATCH] [REVERTME] Patch to pass None for GetBootImagePropFile

This is a work around patch to solve the build error.
This needs to be reverted and fixed properly.

Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Change-Id: I8a7028b393d0688d2b57fcb749a888de9068638a
---
 tools/releasetools/common.py | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index facde74365..73c383eb8c 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -872,7 +872,9 @@ class PartitionBuildProps(object):
     except KeyError:
       logger.warning('Failed to read IMAGES/boot.img')
       return ''
-    prop_file = GetBootImageBuildProp(boot_img, ramdisk_format=ramdisk_format)
+    prop_file = None
+    """GetBootImageBuildProp(boot_img, ramdisk_format=ramdisk_format)"""
+    logger.warning('Returning None for build.prop for boot image')
     if prop_file is None:
       return ''
     with open(prop_file, "r") as f:
@@ -3700,7 +3702,8 @@ def GetBootImageTimestamp(boot_img):
     if file has unknown format. Raise exception if an unexpected error has
     occurred.
   """
-  prop_file = GetBootImageBuildProp(boot_img)
+  prop_file = None
+  """GetBootImageBuildProp(boot_img)"""
   if not prop_file:
     return None
 
-- 
2.17.1

